internal/pkg/term/log/log.go

```diff
@@ -6,10 +6,17 @@ package log
 
 import (
 	"fmt"
+	"io"
 
 	"github.com/fatih/color"
 )
 
+// Wrapper writers around standard error and standard output that work on windows.
```

Comment:
I know this comment existed before, but would this change make it make more sense?
```suggestion
// Wrap writers around standard error and standard output that work on windows.
```

Reply:
oh sorry these are meant to be "wrapper" like decorated writers.
I'll update the comment so it's not confusing!